plugins/web/opentelemetry-instrumentation-document-load/src/utils.ts

```diff
@@ -22,6 +22,7 @@ import {
   PerformanceLegacy,
   PerformanceTimingNames as PTN,
 } from '@opentelemetry/sdk-trace-web';
+import { getCLS, getFCP, getFID, getLCP, getTTFB, Metric } from 'web-vitals'
```

Comment:
What happened to semicolons, here & in many other places? 😄 BTW, I'd suggest setting up Prettier in the repo.

Reply:
Fixed.